recipes/plumpy/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "plumpy" %}
+{% set version = "0.7.9"%}
```

Comment:
Yes, I have just [done](https://github.com/conda-forge/aldjemy-feedstock/pull/1) that for aldjemy 0.7.0.
Just to be sure: If the tests pass, I merge the pull request by myself, right?